scripts/release.sh

```diff
@@ -53,13 +53,24 @@ fi
 
 # Build Docker server image
 if [[ -z "$SKIP_DOCKER" ]]; then
-  docker build -t $docker_name:$version .
-  docker build -t $docker_name:$version-standalone --build-arg STANDALONE=1 .
+  docker build -t $docker_name:$version -f amd64.Dockerfile .
+  docker build -t $docker_name:$version-standalone-amd64 -f amd64.Dockerfile --build-arg STANDALONE=1 .
+  docker build -t $docker_name:$version-standalone-arm32v7 -f arm32v7.Dockerfile .
+
+  # We need to create the multi arch image for -standalone
+  # Make sure experimental docker cli feature is on: echo "{ \"experimental\": \"enabled\" }" >> $HOME/.docker/config.json
```

Comment:
I was not familiar with this, very cool! This means that users on arm32v7 could install `shesek/spark-wallet:standalone` and automatically get the appropriate version? Neat. :)

Reply:
Yes, all images used by BTCPay rely on this for multi arch (https://github.com/btcpayserver/btcpayserver-docker#support)

You can see if an image on docker hub is a manifest by doing `docker manifest redis` for example. It will list supported architectures.